### DISCUSSION TITLE
Table Column Width Hides Fields After After ~20 Characters, even when space

### DISCUSSION THREAD

**Discussion by CosmasSC128** [Budibase/budibase#5177] [Help] [answered]
The tables are not very helpful for displaying data if the table is 1200px wide, with only 4 fields, but the data is concealed automatically after 20-25 characters.

Is there a way to change this?

Please let me know if anyone has a way.

edit: it looks like columns have a hardcoded width of 150px in CSS. I just want to remove this or change to a percent (calc% etc).

**Comment by mjashanks** [comment, marked as answer]
Hey @CosmasSC128 

You should be able to do this on the latest release. We recently added the ability to configure column widths....

1. Click "Configure Columns:

![image](https://user-images.githubusercontent.com/3524181/160923271-cc728ead-54c0-4f51-9025-2701153a3ac3.png)

2. Click on the settings icons, beside the column

![image](https://user-images.githubusercontent.com/3524181/160923364-15ebdef8-2bee-4c2d-b56b-6f84bb02aca6.png)

3. Save the settings

**Reply by CosmasSC128** [reply]
Ah, missed the new release features. I'll keep a better eye on it on the future. Thank you!
